src/users/interfaces/remove-user-from-organization-options.interface.ts

```diff
@@ -0,0 +1,4 @@
+export interface RemoveUserFromOrganizationOptions {
+  id: string;
```

Comment:
Same here, since there are two ids.

```suggestion
  user_id: string;
```